client/src/components/ManageResourcesTable.jsx

```diff
@@ -0,0 +1,146 @@
+// @flow
+
+import React, { useState, useEffect } from 'react';
+import { EditFilled } from '@ant-design/icons';
+import { Table, Tag } from 'antd';
+
+import { getResources } from '../utils/api';
+
+const TAG_COLOR_DICT = {
```

Comment:
Might be better to put this in `src/utils` file and then import the dictionary just to clean the file. But definitely a design choice.